Makefile

```
@@ -240,10 +240,19 @@ binary-numacell: build-tools ## Build the numacell test device plugin binary.
 	CGO_ENABLED=0 go build -mod=vendor -o bin/numacell -ldflags "$$LDFLAGS" test/deviceplugin/cmd/numacell/main.go
 
 .PHONY: binary-getdigests
-binary-getdigests: 
+binary-getdigests: ## Build the getdigests helper binary.
 	LDFLAGS="-s -w"; \
 	go build -mod=vendor -o bin/getdigests -ldflags "$$LDFLAGS" tools/getdigests/getdigests.go
 
+.PHONY: update-scheduler-digests
+update-scheduler-digests: binary-getdigests ## Refresh embedded scheduler image digests from the production registry.
```

checkmake (0.3.2)

[warning] Target body for "update-scheduler-digests" exceeds allowed length of 5 lines (6).

(maxbodylength)